.mvn/develocity.xml

```diff
@@ -19,25 +19,22 @@
     under the License.
 
 -->
-<gradleEnterprise
-        xmlns="https://www.gradle.com/gradle-enterprise-maven" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-        xsi:schemaLocation="https://www.gradle.com/gradle-enterprise-maven https://www.gradle.com/schema/gradle-enterprise-maven.xsd">
-  <!-- Enable Gradle Enterprise extension when GRADLE_ENTERPRISE_ACCESS_KEY is set and the build isn't
+<develocity xmlns="https://www.gradle.com/develocity-maven" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="https://www.gradle.com/develocity-maven https://www.gradle.com/schema/develocity-maven.xsd">
+<!-- Enable Gradle Enterprise extension when GRADLE_ENTERPRISE_ACCESS_KEY is set and the build isn't
```

Comment:
should we update this comment (e.g. Enable Develocity ...)?

Reply:
I'll do that later, once ASF Infra has switched to use DEVELOCITY_ACCESS_KEY.

Reply:
Updated